src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/CompletenessFilterIntegration.php

```diff
@@ -328,7 +402,7 @@ public function testErrorLocalesIsMalformed()
 
     /**
      * @expectedException \Akeneo\Component\StorageUtils\Exception\InvalidPropertyException
-     * @expectedExceptionMessage Property "completeness" expects a valid scope.
```

Comment:
nope, we will have problem on API if you change scope to channel.
PO wanted to have "scope" everywhere to be consistent